### PR TITLE
Feature/online outfit memmber logging

### PR DESCRIPTION
**Pull request by karlis-ozo**
Based on configured list of outfits, this monitors and saves maximum amount of people online once every 5 minutes. but the frequency can be easily changed later.
```json
{
  "timestamp": "2022-10-12T18:10:34.704+00:00",
  "metadata": {
    "outfit":  37509438620602936
  },
  "player_count": 21,
  "_id":  "6347004211f51da570f634d4"
}
```
it also looks like another Mega PR is going to master. although we could move arma logging out of the bot as that is in no way related or needs to be configured by the bot

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-version: "3"
+version: "3.x"
```
Comment (FrankGrimm) [inline docker-compose.yml:1]: My version of docker-compose doesn't support `.x` in here, mind setting this directly to `3.9`?

File: src/loggers/ps2_outfit_logger.py [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+        initializes ps2 logger.
+        Parameters
+        ----------
+        db: Database class instance to use
+        saving_period: time in seconds for fow frequently the data should be saved
+        """
+        self.saving_period: float = saving_period
+        FU_id = 37509488620602936
+        nFUc_id = 37558455247570544
+        vFUs_id = 37558804429669935
```
Comment (FrankGrimm) [inline src/loggers/ps2_outfit_logger.py:30]: Hardcoded IDs in the middle of the code aren't ideal, they are needed elsewhere anyway right? Separate python file or a config file for the monitored outfits maybe?

File: docker-compose.yml (hunk 1 above)
Comment (karlis-ozo) [reply docker-compose.yml]: I have set the version to be 3.9. As well as removed port mapping as that is not needed at this moment.

File: src/loggers/ps2_outfit_logger.py [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+    ):
+        """
+        initializes ps2 logger.
+        Parameters
+        ----------
+        db: Database class instance to use
+        saving_period: time in seconds for fow frequently the data should be saved
+        """
+        self.saving_period: float = saving_period
+        FU_id = 37509488620602936
```
Comment (Normandabald) [inline src/loggers/ps2_outfit_logger.py:28]: Could these be a list map that we get from an environment variable? Might make it much easier to add/remove Outfits to track in the future

File: src/loggers/ps2_outfit_logger.py [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+        async def logged_in(_event: event.PlayerLogin):
+            character = await client.get_by_id(ps2.Character, _event.character_id)
+            name: str = character.data.name.first
+            outfit = await character.outfit()
+            if not outfit:
+                return
+            if outfit.id in self._monitored_outfits:
+                await self._save_player_count(outfit.id, await census.get_online_outfit(outfit.id))
+            logging.info(
+                f'{name} logged in at {_event.timestamp} and his outfit is {outfit.tag if outfit is not None else "non existent"} ')
```
Comment (Normandabald) [inline src/loggers/ps2_outfit_logger.py:107]: Gendered logging? Tut tut, it's 2022 don't you know!

`[{Outfit Tag}] {Name} logged in`

**Review by Normandabald — CHANGES_REQUESTED** [review]
Looks good! Just some minor suggestions

**Pull request merged**